templates/render

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'erb'
+require 'json'
+require 'base64'
+
+redis_result = JSON.pretty_generate(YAML.load(ERB.new(File.read("../services/redis.yml.erb")).result))
```

Comment:
Use an array to loop through all the erbs so that someone will not have to change the code for new services addition.